<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitCloneDialog</class>
 <widget class="QDialog" name="GitCloneDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>390</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_local">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="  When the repository to clone from is on a local machine, this flag    bypasses the normal &quot;Git aware&quot; transport mechanism and clones the    repository by making a copy of HEAD and everything under objects and    refs directories. The files under .git/objects/ directory are    hardlinked to save space when possible. "/>
   </property>
   <property name="text">
    <string>-l, --local</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_no_hardlinks">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>--no-hardlinks</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_shared">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-s, --shared</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_reference">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>--reference &lt;repository&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_dissociate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>--dissociate</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_quiet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-q, --quiet</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_verbose">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-v, --verbose</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_progress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>--progress</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_no_checkout">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-n, --no-checkout</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_bare">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>--bare</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_mirror">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>--mirror</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_origin">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-o, --origin &lt;name&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_branch">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-b, --branch &lt;name&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_uploadpack">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-u, --upload-pack &lt;upload-pack&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_template">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>--template=&lt;template_directory&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_config">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-c, --config &lt;key&gt;=&lt;value&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_depth">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>--depth &lt;depth&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_single_branch">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>--[no-]single-branch</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_recursive">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>--recursive, --recurse-submodules</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_separate_git_dir">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>--separate-git-dir=&lt;git dir&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_directory">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_directory">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>directory</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>20</width>
     <height>311</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GitCloneDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GitCloneDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
